src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -383,17 +388,25 @@ export default React.memo(function GamePage(): JSX.Element | null {
                     >
                       <b>{t('info.path')}:</b> {appLocation}
                     </div>
-                    {isLinux && !isNative && (
+                    {!isWin && !isNative && (
                       <>
-                        <div>
-                          <b>Wine:</b> {wineVersion}
-                        </div>
-                        <div
-                          className="clickable"
-                          onClick={() => window.api.openFolder(winePrefix)}
-                        >
-                          <b>Prefix:</b> {winePrefix}
-                        </div>
+                        <b>Wine:</b> {wineVersion?.name}
+                        {wineVersion && wineVersion?.type === 'crossover' ? (
```

Comment:
`wineVersion` can't be `undefined` if you're checking it for truthy-ness first, so
```suggestion
                        {wineVersion && wineVersion.type === 'crossover' ? (
```
should be fine

Reply:
Ok. Wont commit your suggestion because Gihub always messes up the lint when I do that, but I will change and push it.